src/components/pages/Data.tsx

```diff
@@ -1,18 +1,10 @@
 import React, { useState } from "react";
 import { useMediaQuery } from "react-responsive";
 import { Link } from "react-router-dom";
-import {
-  Segment,
-  Button,
-  Accordion,
-  Icon,
-  AccordionTitleProps,
-} from "semantic-ui-react";
-import { isMaintenanceMode, mobileDeviceOrTabletWidth } from "../../constants";
+import { Segment, Button, Accordion, Icon, AccordionTitleProps } from "semantic-ui-react";
+import { isMaintenanceMode, mobileDeviceOrTabletWidth, PAGE_HASHES } from "../../constants";
```

Comment:
What exactly is PAGE_HASHES?

Reply:
These are the Identifiers used in links in the footer. Since these links are not just to a page, but a certain section of the page, we use these `PAGE_HASHES` as ways to identify the sub sections and link everything together